taskiq_pipelines/middleware.py

```diff
@@ -41,9 +41,21 @@ async def post_save(  # noqa: C901, WPS212
             logger.warn("Pipline data not found. Execution flow is broken.")
             return
         pipeline_data = message.labels[PIPELINE_DATA]
+        # workaround for obligatory casting label values to `str`
+        # in `AsyncKicker._prepare_message`.
+        # The trick can be removed later after adding explicit `bytes` support.
+        if (  # noqa: WPS337
+            isinstance(pipeline_data, str)
```

Comment:
I'm wondering is this because of this line:

https://github.com/taskiq-python/taskiq/blob/a74fa4d4bc47662ea46dc3af51c156fc23c21d94/taskiq/kicker.py#L253

Maybe before merging we can add support for bytes in labels, by removing this str modifier? Since I really don't like how it looks. Guess removig str should give no effect, bu might break compatibility. I will find out how to make it possible without big compatibility problems.

Reply:
Yes, exactly. It is the result of `str(label)` conversion.

I suspect that the plain `str(...)` removal is not enough.  Let me experiment first and come up with a PR for taskiq core project.

Reply:
Yes. I tried removing it and broke tests.